src/Columns/ColumnVector.cpp

```diff
@@ -123,11 +124,19 @@ namespace
 }
 
 template <typename T>
-void inplacePartialSwapHelper(T& arr, size_t bound)
+void shiftArrayRight(T& arr, size_t bound)
```

Comment:
Is it better than https://en.cppreference.com/w/cpp/algorithm/rotate ?

Reply:
Doubtful